applications/graphs/tests.py

```diff
@@ -37,7 +37,7 @@ def test_crud_operation(self):
 
 		# Create
 		self.session.add(User(email='owner@example.com', password="password", is_admin=0))
-		self.session.add(Graph(name='graph1', owner_email='owner@example.com', json='{}', is_public=0))
```

Comment:
Removed 'json' parameter as we are no longer storing graph_json and style_json in Graph Table.